layers/meta-balena-allwinner/recipes-kernel/linux/linux-mainline_%.bbappend

```diff
@@ -22,11 +22,22 @@ RESIN_CONFIGS[axp_power] ="\
 
 RESIN_CONFIGS_append_orangepi-plus2 = " wifi"
 RESIN_CONFIGS_append_orange-pi-zero = " wifi"
+RESIN_CONFIGS_append_orange-pi-lite = " wifi wlan"
+RESIN_CONFIGS_append_nanopi-neo-air = " wifi wlan"
+RESIN_CONFIGS_append_bananapi-m1-plus = " wifi wlan"
+
 RESIN_CONFIGS[wifi] ="\
     CONFIG_WIRELESS=y \
     CONFIG_RFKILL=y \
     CONFIG_CFG80211=m \
     CONFIG_CFG80211_WEXT=y \
+    CONFIG_WLAN=y \
+    CONFIG_WLAN_VENDOR_REALTEK=y \
+"
+
+RESIN_CONFIGS[wlan] ="\
+    CONFIG_WLAN=y \
+    CONFIG_WLAN_VENDOR_REALTEK=y \
```

Comment:
Why are these duplicated?

Reply:
Removed duplicates

Reply:
Let's also move them to the `wifi` group from above